<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceListWidget</class>
 <widget class="QGroupBox" name="ResourceListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Resources</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QListWidget" name="resourceList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bAddResource">
       <property name="text">
        <string>Add resource...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bEditResource">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Modify resource</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bDelResource">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove resource</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
